tests/raw_to_standard/planned_events_test.py

```diff
@@ -0,0 +1,222 @@
+                }
+            ],
+            "direction": "east"
+        }
+    }
+
+    actual = planned_events.expand_event_directions(event)
+    print(actual)
+
+    assert expected_results.test_expand_speed_zone_1_expected == actual
+
+
+def test_expand_event_directions_2():
+    event = {
+        "type": "Feature",
+        "geometry": {
+            "srid": 4326,
+            "type": "MultiPoint",
+            "coordinates": [
+                [
+                    -108.279106,
+                    39.195663
+                ],
+                [
+                    -108.218549,
+                    39.302392
+                ]
+            ]
+        },
+        "properties": {
+            "clearTime": "2022-05-01T18:26:04.000+00:00",
+            "startMarker": 50.0,
+            "type": "Bridge Construction",
+            "laneImpacts": [
+                {
+                    "direction": "east",
+                    "laneCount": 2,
+                    "laneClosures": "6000",
+                    "closedLaneTypes": [
+                        "left lane",
+                        "right lane"
+                    ]
+                }
+            ],
+            "routeName": "I-70E",
+            "isOversizedLoadsProhibited": True,
+            "lastUpdated": "2021-10-29T18:35:01.835+00:00",
+            "schedule": [
+                {
+                    "startTime": "2021-10-29T18:26:04.000Z",
+                    "endTime": "2022-05-01T18:26:04.000Z"
+                }
+            ],
+            "endMarker": 60.0,
+            "startTime": "2021-10-29T18:26:04.000+00:00",
+            "id": "OpenTMS-Event1689408506",
+            "travelerInformationMessage": "Between Exit 49: CO 65; Grand Mesa (5 miles east of the Palisade area) and US 6 (Debeque) from Mile Point 50 to Mile Point 60. Road closed expect delays due to bridge construction. Until May 1, 2022 at about 12:26PM MDT.",
+            "direction": "east"
+        },
+        "attributes": {}
+    }
+
+    actual = planned_events.expand_event_directions(event)
+
+    assert expected_results.test_expand_speed_zone_2_expected == actual
+
+
+@patch('uuid.uuid4')
+def test_generate_standard_messages_from_string(mockuuid):
+    uuid.uuid4 = Mock()
+    uuid.uuid4.side_effect = ['we234de', '23wsg54h']
+    actual_standard = json.loads(json.dumps(planned_events.generate_standard_messages_from_string(
+        expected_results.test_generate_standard_messages_from_string_input)))
+    for i in actual_standard:
+        del i['rtdh_timestamp']
+        del i['event']['source']['last_updated_timestamp']
+    expected = expected_results.test_generate_standard_messages_from_string_expected
+    for i in expected:
+        del i['rtdh_timestamp']
+        del i['event']['source']['last_updated_timestamp']
+    # actual_standard = [dict(x) for x in actual_standard]
+    print(actual_standard)
+    assert actual_standard == expected
+
+
+def test_get_lanes_list():
```

Comment:
This test is named the same as the one below it, so only one of them runs. Do we need to run both?

Reply:
renamed